### PR TITLE
update stale bot config

### PR DESCRIPTION
**Pull request by arvidn**

### CODE REVIEW

**Comment by AllSeeingEyeTolledEweSew** [comment]
This doesn't address my largest complaint about stalebot, which is that it has apparent bugs, and will close an issue despite activity. See #5154 and #6198

**Comment by AllSeeingEyeTolledEweSew** [comment]
This doesn't address my largest complaint about stalebot, which is that it has apparent bugs, and will close an issue despite activity. See #5154 and #6198

File: .github/stale.yml [hunk 1]
```diff
@@ -1,11 +1,12 @@
...
-daysUntilStale: 90
+daysUntilStale: 120
 # Number of days of inactivity before a stale issue is closed
-daysUntilClose: 20
+daysUntilClose: 30
 # Issues with these labels will never be considered stale
 exemptLabels:
   - pinned
   - security
+  - todo
```
Comment (AllSeeingEyeTolledEweSew) [inline .github/stale.yml:9]: the effect of this is that I'll just prompt you to add `todo` to any issue I open

Comment (AllSeeingEyeTolledEweSew) [inline .github/stale.yml:9]: the effect of this is that I'll just prompt you to add `todo` to any issue I open

Comment (arvidn) [reply .github/stale.yml:9]: that's not an obvious problem to me.

**Comment by arvidn** [comment]
I should research whether there are some configuration options of stale bot to improve or fix this. Perhaps it just has the wrong default.

File: .github/stale.yml (hunk 1 above)
Comment (AllSeeingEyeTolledEweSew) [reply .github/stale.yml:9]: it seems backwards that high quality reports should carry the burden of requesting the "good issue" label, arguing why they deserve it, knowing to do so in the first place, re-requesting it if you forget to apply the label when responding to the ticket, and dealing with the additional noise.

I make high quality reports and I want all the burden to be on low-quality reporters :)

**Comment by summerqB** [comment]
428 issues closed by stale bot 😮

**Comment by userdocs** [comment]
Most of my issues closed by stale bot have been generic build stuff I forgot to close or was never resolved but became obsolete from using newer versions and stuff so I can't say it caused me any drama but I think it should work like this:

Stale bot should message the issue and ask the op if they want to keep it open and if not can they please tag it can close it manually.

So basically interact with the OP act in response to that outcome.

Maybe after x amount of non responses default to a behaviour.

I think using it to encourage the OP manage the issue is the best first step. At least that way they wont just get their issue closed?

Maybe stale bot forces someone to tag the issue with a easily filterable tag before closing so relevant issues can be filtered when searching? Good for python binding specific issues for example.

Dead stuff with no response will get closed for lack of reply and

**Pull request closed without merging**